stylelint-polaris/plugins/global-disallowed-list/index.test.js

```diff
@@ -0,0 +1,30 @@
+/* global testRule */
+
+const {messages, ruleName} = require('.');
+
+const config = ['--p-button-font'];
+
+testRule({
+  ruleName,
+  plugins: [__dirname],
+  config,
+  customSyntax: 'postcss-scss',
+  accept: [
+    {
+      code: '.valid { color: red; }',
+      description: 'Uses nothing on the banned list',
```

Comment:
```suggestion
      description: 'Uses nothing on the deny list',
```
Light suggestion to use `allow|deny list` convention

Reply:
I went with "disallowed list" to match the rule name